### DISCUSSION TITLE
Reference required

### DISCUSSION THREAD

**Discussion by kasheex** [CURENT/andes#549] [Modeling]
Please provide the reference to convert transfer function $\frac{y}{u} = K \frac{1+sT_1}{1+sT_2}$ into the implementable equations $$T_2 \dot{x}^\prime = (u - x^\prime)$$  
$$T_2 y = KT_1(u-x^\prime) + KT_2x^\prime +E_2$$

**Comment by jinningwang** [comment]
The `LeadLag` block should be ready to use

https://github.com/CURENT/andes/blob/5ab784b75347a98d07c004e32d7c4b39d407ea2e/andes/core/block.py#L1410-L1494
